CONTRIBUTORS.md

```diff
@@ -0,0 +1,111 @@
+## Open-Source Collaboration Best Practices
+Please refer yourself to the [best practices document](https://portal.collab.admin.ch/sites/602-eak-copilot/_layouts/15/WopiFrame.aspx?sourcedoc=%7B7AC07C75-D5A2-4107-BE9C-EF376BCCE416%7D&file=Open-Source%20project%20Collaboration%20%E2%80%93%20Best%20Practices%20Updated.pptx&action=default&IsList=1&ListId=%7BCDF81277-FBCD-4EB8-B449-D6E0BE5A98C9%7D&ListItemId=21) for guidelines on how to contribute to this project.
```

Comment:
We shouldn't share from the internal sharepoint, I'm not sure if PowerPoint is the best, maybe we convert it to a separate md file or we link it in the documentation:
[gpt-4o PowerPoint to md](https://chat.openai.com/share/6afbbd50-e646-4617-b201-f1c4988d87bf)

Reply:
I've transcript the best pratice ppt in an .rst file to put it in the documentation website, haven't pushed it yet.

Reply:
https://cdc-si.github.io/eak-copilot/guidelines/opensource.html

Reply:
@Shi-Ho That's great thanks!